README.md

```diff
@@ -2,15 +2,6 @@
 
 A kde plasma accent color changer that changes some colors in current colorscheme.
 
-## Getting Started!
+Please Please accept my MR
```

Comment:
YOU added this line. In a real MR developers would have yelled at you for some reason, if it was some real code. That's how most of the reviews are done.

Reply:
Your project has copy pasted code

Reply:
Okay